.github/workflows/push_rockspec.yml

```diff
@@ -33,15 +33,13 @@ jobs:
 
       # https://stackoverflow.com/questions/58177786/get-the-current-pushed-tag-in-github-actions
       - name: Set env
-        run: echo "GIT_TAG=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+        run: echo "GIT_TAG=${GITHUB_REF##*/}" >> $GITHUB_ENV
 
       - name: Push release rockspec
         run: |
-          sed \
-            -e "s/branch = '.\+'/tag = '${GIT_TAG}'/g" \
-            -e "s/version = '.\+'/version = '${GIT_TAG}-1'/g" \
-            ${{ env.ROCK_NAME }}-scm-1.rockspec > ${{ env.ROCK_NAME }}-${GIT_TAG}-1.rockspec
-          sed -i "s/local VERSION = 'scm-1'/local VERSION = '"${GIT_TAG}"-1'/g" metrics/init.lua
+          sed -i "s/local VERSION = 'scm-1'/local VERSION = '"${env.GIT_TAG}"-1'/g" metrics/init.lua
+      - run: tarantoolctl rocks install ${{ env.ROCK_NAME }}-${{ env.GIT_TAG }}-1.rockspec
+      - run: tarantoolctl rocks pack ${{ env.ROCK_NAME }} ${{ env.GIT_TAG }}
```

Comment:
Here we create an `.all.rock` file and **don't** upload it. So the module with patched version string will not present on the rock server.

I'm not sure that `rocks install` will install the module with patched version string. I guess it'll do a fresh clone of the repository pointed by `source.{url,branch,tag}`. Did you check it? I use `rocks make` for this purpose. You can find background information here: https://github.com/tarantool/graphql/commit/029fcf2f0475024db8629e270d613a2da12b2a4b.